packages/@o3r/testing/schematics/ng-add/schema.json

```diff
@@ -2,14 +2,31 @@
   "$schema": "http://json-schema.org/schema",
   "$id": "ngAddSchematicsSchema",
   "title": "Add Otter testing ",
-  "description": "ngAdd Otter testing ",
+  "description": "ngAdd Otter testing",
   "properties": {
     "projectName": {
       "type": "string",
       "description": "Project name",
       "$default": {
         "$source": "projectName"
       }
+    },
+    "testingFramework": {
+      "type": "string",
+      "description": "The testing framework to use",
+      "default": "jest",
+      "enum": [
+        "jest",
+        "other"
+      ],
+      "x-prompt": {
```

Comment:
we don't need to ask this question if we can detect that the user depends on jest 
we can take the assumption that he will use jest
So I will ask the question in the `index.ts` if the project doesn't depends on `jest`

Reply:
The goal is that, eventually, we will setup Jest (jest config, needed deps, scripts in package.json etc) via the ng-add of the testing module